MANUAL_SETUP.md

```diff
@@ -0,0 +1,162 @@
+# Setup for v6 React Native Storybook
+
+Before getting into the guide consider using a template for a simpler setup process.
+
+**Prebuilt Templates:**
+
+For expo you can use this [template](https://github.com/dannyhw/expo-template-storybook) with the following command
+
+```sh
+expo init --template expo-template-storybook AwesomeStorybook
+```
+
+For react native cli you can use this [template](https://github.com/dannyhw/react-native-template-storybook)
+
+```sh
+npx react-native init MyApp --template react-native-template-storybook
+```
+
+You can also choose to use a bash script if you prefer, however it is more involved.
+
+- [rn-cli setup script](https://gist.github.com/dannyhw/9b84973dcc6ff4fa2e86e32d571d294e)
+
+- [expo setup script](https://gist.github.com/dannyhw/92b3ff0d6ccaead9df2820a507154b87)
+
+# Manual setup
+
+You may wish to setup everything yourself, you can use the following guide to do so.
+
+# Install dependencies
+
+**React native CLI**
```

Comment:
Do you think you can follow the same order as Prebuilt?

**Prebuilt**
- expo
- react-native cli

**install deps**
- react-native cli
- expo

I would swap to 

**install deps**
- expo
- react-native cli

Reply:
That's a good point, makes it more consistent. Thanks!